<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientFrame</class>
 <widget class="QFrame" name="PatientFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>535</height>
   </rect>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="patientBox">
     <property name="title">
      <string notr="true">Patient search</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lastNameLabel">
          <property name="toolTip">
           <string comment="PATIENT LAST SEARCH TOOLTIP">PATIENT LAST</string>
          </property>
          <property name="text">
           <string comment="PATIENT LAST SEARCH LABEL">PATIENT LAST</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lastNameField">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string comment="PATIENT LAST SEARCH TOOLTIP">PATIENT LAST</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="firstNameLabel">
          <property name="toolTip">
           <string comment="PATIENT FIRST SEARCH TOOLTIP">PATIENT FIRST</string>
          </property>
          <property name="text">
           <string comment="PATIENT FIRST SEARCH LABEL">PATIENT FIRST</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="firstNameField">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string comment="PATIENT FIRST SEARCH TOOLTIP">PATIENT FIRST</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="dobLabel">
          <property name="toolTip">
           <string comment="DOB SEARCH TOOLTIP">DOB</string>
          </property>
          <property name="text">
           <string comment="DOB SEARCH LABEL">DOB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dobField">
          <property name="toolTip">
           <string comment="DOB SEARCH TOOLTIP">DOB</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="activeField">
          <property name="text">
           <string notr="true">Active</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string notr="true">Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string notr="true">Search</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="newPatientButton">
       <property name="text">
        <string notr="true">New Patient</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout2">
     <item>
      <widget class="QPushButton" name="prescribeButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string notr="true">New Prescription</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="modifyButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string notr="true">Modify Patient</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="DisplayTable" name="resultTable">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>32</number>
     </attribute>
     <column>
      <property name="text">
       <string>patients.id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="ALLSCRIPTS ID DISPLAY LABEL">ALLSCRIPTS ID</string>
      </property>
      <property name="toolTip">
       <string comment="ALLSCRIPTS ID DISPLAY TOOLTIP">ALLSCRIPTS ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="PATIENT LAST DISPLAY LABEL">PATIENT LAST</string>
      </property>
      <property name="toolTip">
       <string comment="PATIENT LAST DISPLAY TOOLTIP">PATIENT LAST</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="PATIENT FIRST DISPLAY LABEL">PATIENT FIRST</string>
      </property>
      <property name="toolTip">
       <string comment="PATIENT FIRST DISPLAY TOOLTIP">PATIENT FIRST</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="DOB DISPLAY LABEL">DOB</string>
      </property>
      <property name="toolTip">
       <string comment="DOB DISPLAY TOOLTIP">DOB</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="PATIENT ACTIVE DISPLAY LABEL">PATIENT ACTIVE</string>
      </property>
      <property name="toolTip">
       <string comment="PATIENT ACTIVE DISPLAY TOOLTIP">PATIENT ACTIVE</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="prescribeAction">
   <property name="text">
    <string notr="true">New Prescription</string>
   </property>
   <property name="toolTip">
    <string notr="true">Prescribe a medication</string>
   </property>
  </action>
  <action name="modifyAction">
   <property name="text">
    <string notr="true">Modify Patient</string>
   </property>
   <property name="toolTip">
    <string notr="true">Modify demographic information</string>
   </property>
  </action>
  <action name="newPatientAction">
   <property name="text">
    <string notr="true">New Patient</string>
   </property>
   <property name="toolTip">
    <string notr="true">Add a new patient</string>
   </property>
  </action>
  <action name="toggleAction">
   <property name="text">
    <string notr="true">Toggle active</string>
   </property>
   <property name="toolTip">
    <string notr="true">Toggle active/inactive</string>
   </property>
  </action>
  <action name="viewAction">
   <property name="text">
    <string notr="true">View Patient</string>
   </property>
   <property name="toolTip">
    <string notr="true">View the patient's record</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisplayTable</class>
   <extends>QTableWidget</extends>
   <header>displaytable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchButton</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>newPatientButton</tabstop>
  <tabstop>prescribeButton</tabstop>
  <tabstop>modifyButton</tabstop>
  <tabstop>resultTable</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>firstNameField</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lastNameField</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>firstNameField</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>lastNameField</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>prescribeButton</sender>
   <signal>clicked()</signal>
   <receiver>prescribeAction</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modifyButton</sender>
   <signal>clicked()</signal>
   <receiver>modifyAction</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>816</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newPatientButton</sender>
   <signal>clicked()</signal>
   <receiver>newPatientAction</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
